### PR TITLE
fernando gomez increment/decrement react app

### PR DESCRIPTION
**Pull request by shd8**

### CODE REVIEW

File: fernando-gomez/challenge/first-react-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import logo from './logo.svg';
+import './App.css';
+import React, { useState } from 'react';
+
```
Comment (JDSanchon) [inline fernando-gomez/challenge/first-react-app/src/App.js:4]: Cada componente debería estar fuera/ independiente¿?

**Review by JDSanchon — DISMISSED** [review]

File: fernando-gomez/challenge/first-react-app/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    node: true,
```
Comment (fredyzz) [inline fernando-gomez/challenge/first-react-app/.eslintrc.js:5]: jest?

File: fernando-gomez/challenge/first-react-app/public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    -->
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
```
Comment (fredyzz) [inline fernando-gomez/challenge/first-react-app/public/index.html:25]: This comments are necessary??

File: fernando-gomez/challenge/first-react-app/src/App.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  <button className="incrementorButton" onClick={incrementor}>
+    +
+  </button>
+);
+
+const Decrementor = ({ decrementor }) => (
+  <button className="incrementorButton" onClick={decrementor}>
+    -
+  </button>
+);
```
Comment (fredyzz) [inline fernando-gomez/challenge/first-react-app/src/App.js:15]: Please, move the components to separated files

**Review by fredyzz — CHANGES_REQUESTED** [review]

**Review by fredyzz — APPROVED** [review]

**Review by annasm07 — APPROVED** [review]
Porque no he mirao eh

**Review by Emerson-feral — APPROVED** [review]

**Pull request merged**